src/preamble.js

```diff
@@ -122,16 +122,6 @@ function assert(condition, text) {
 
 var globalScope = this;
 
-// Returns the C function with a specified identifier (for C++, you need to do manual name mangling)
-function getCFunc(ident) {
-  var func = Module['_' + ident]; // closure exported function
-  if (!func) {
```

Comment:
to limit the risk of surprising users, how about keeping `getCFunc` and just removing the 3 lines with the if and the eval? then it would show an error message when the function can't be found.

also, please change the error message (a few lines down in the assert) to something like `Cannot call unknown function ' + ident + ', make sure it is exported`.

Reply:
Good point! Done.